sway/input/seat.c

```diff
@@ -73,6 +73,32 @@ static void seat_node_destroy(struct sway_seat_node *seat_node) {
 	free(seat_node);
 }
 
+void seat_notify_key_press(struct sway_seat *seat) {
```

Comment:
Hm, I think I'd prefer to leave this in `cursor.c`, alongside the current hide-cursor-after-timeout implementation.

Reply:
(This function can take a `sway_cursor` instead of a `sway_seat` I believe.)

Reply:
Done